apps/deploy-web/src/components/layout/Nav.tsx

```diff
@@ -24,9 +25,14 @@ export const Nav = ({
   const theme = useCookieTheme();
   const [isSignedInWithTrial] = useAtom(walletStore.isSignedInWithTrial);
   const { user } = useCustomUser();
+  const hasCreditCardBanner = useHasCreditCardBanner();
 
   return (
-    <header className="fixed top-0 z-50 w-full border-b border-border bg-popover dark:bg-background">
+    <header
+      className={cn("fixed top-0 z-50 w-full border-b border-border bg-popover dark:bg-background", {
+        "top-[40px]": hasCreditCardBanner
```

Comment:
so now both layout and nav have a dep on this banner. This here should be accepting a `className` prop and be controlled on the parent.